src/reducer.js

```diff
@@ -24,6 +24,7 @@ import viewport from './viewport/reducer'
 import expandedView from './expanded-view/reducer'
 import goalVotes from './goal-votes/reducer'
 import goalComments from './goal-comments/reducer'
+import goalClone from './goal_clone/reducer'
```

Comment:
I know this seems trivial, but can you make the `goal_clone` folder `goal-clone`
Sorry!